Structure_Engine/Query/LocalOrientation.cs

```diff
@@ -57,7 +57,7 @@ public static Basis LocalOrientation(this Panel panel)
         [Output("orienations", "The local orientations of the mesh faces as a list of vector Bases.")]
         public static List<Basis> LocalOrientations(this FEMesh mesh)
         {
-            return mesh.NullCheck("LocalOrientations", false, false) ? mesh.Faces.Select(x => x.LocalOrientation(mesh)).ToList() : null;
+            return mesh.IsNull(false, false) ? null : mesh.Faces.Select(x => x.LocalOrientation(mesh)).ToList();
```

Comment:
Not sure why this is using false for if nodes and faces should be checked? they are required, no?